src/Enum.php

```diff
@@ -165,6 +165,19 @@ public static function search($value)
         return array_search($value, static::toArray(), true);
     }
 
+    /**
+     * Returns the key's value
+     *
+     * @param $key
+     * @return mixed
+     */
+    public static function findValue($key)
+    {
+        $enumArray = static::toArray();
+
+        return @$enumArray[$key] ?: null;
```

Comment:
You should use an isset check instead of suppressing errors.

Reply:
I used isset in my last commit, but still is giving error